example/scatter-plot-changing-shape.md

```diff
@@ -0,0 +1,26 @@
+# Scatter Plot/Bubble Plot Changing shape
+
+`Scatter Plot` defaults to being `Circle`. To change its shape to `Rectangle`, set the `shape` property to `"Rect"`:
+
+```js
+var myData = [
+    {"value": 100, "weight": .45,  "name": "alpha"},
+    {"value": 70,  "weight": .60,  "name": "beta"},
+    {"value": 40,  "weight": -.2,  "name": "gamma"},
+    {"value": 15,  "weight": .1,   "name": "delta"},
+    {"value": 5,   "weight": -.43, "name": "epsilon"},
+    {"value": 1,   "weight": 0,    "name": "zeta"}
+  ]
+
+new d3plus.Plot()
+  .config({
```

Comment:
let's start alphabetically sorting the keys in the config objects, it makes them easier/quicker to find